README.rst

```diff
@@ -64,8 +64,9 @@ Real life example::
   max-line-length = 100
   known-modules = my-lib:[mylib.drm,mylib.encryption]
 
-If you use `flake8-pyproject <https://pypi.org/project/Flake8-pyproject/>`_, you can also configure
-the known modules using a nicer syntax::
+If you use `Flake8-pyproject <https://pypi.org/project/Flake8-pyproject/>`_
+(can include for installation using ``flake8-requirements[toml]``),
```

Comment:
The extras in form `flake8-requirements[toml]` looks more like toml support... How about explicit: `flake8-requirements[pyproject]`? If you are OK with such change, please also bump minor version, so I could make automatic release :)

Reply:
Sure sounds good! Implemented this just now

Let's merge it after https://github.com/arkq/flake8-requirements/pull/86, and give a minor version bump

Reply:
Current form is OK